### PR TITLE
#565 Android api level 16 unsupported

### PR DESCRIPTION
**Pull request by divyanshu29**
I just removed the lines dealing with android api level 16? @whimboo Please have a look at it!

### CODE REVIEW

File: tests/remote/test_fennec.py [hunk 1]
```diff
@@ -19,18 +19,7 @@
     # Support for API level 15 ended on Aug 29th 2017
     {'application': 'fennec', 'platform': 'android-api-15',
      'branch': 'mozilla-central', 'date': '2017-08-29'},
-    {'application': 'fennec', 'platform': 'android-api-16', 'branch': 'mozilla-central'},
     {'application': 'fennec', 'platform': 'android-x86', 'branch': 'mozilla-central'},
-    {'application': 'fennec', 'platform': 'android-api-16',
-     'branch': 'mozilla-central', 'date': '2017-08-30'},
```
Comment (whimboo) [inline tests/remote/test_fennec.py]: Actually please only remove those without a date or build id. As it looks like only the latest links have been removed. Builds from 2017 are still around.

File: tests/remote/test_fennec.py [hunk 2]
```diff
@@ -19,18 +19,7 @@
     # Support for API level 15 ended on Aug 29th 2017
     {'application': 'fennec', 'platform': 'android-api-15',
      'branch': 'mozilla-central', 'date': '2017-08-29'},
-    {'application': 'fennec', 'platform': 'android-api-16', 'branch': 'mozilla-central'},
     {'application': 'fennec', 'platform': 'android-x86', 'branch': 'mozilla-central'},
```
Comment (whimboo) [inline tests/remote/test_fennec.py]: Maybe you can change that line to make use of a date instead like https://archive.mozilla.org/pub/mobile/nightly/2019/05/2019-05-17-16-29-00-mozilla-central-android-x86/

**Comment by Divyanshu29** [comment]
@whimboo Can u plz have a look now? I made the changes as suggested by you!

File: tests/remote/test_fennec.py [hunk 3]
```diff
@@ -19,8 +19,8 @@
     # Support for API level 15 ended on Aug 29th 2017
     {'application': 'fennec', 'platform': 'android-api-15',
      'branch': 'mozilla-central', 'date': '2017-08-29'},
-    {'application': 'fennec', 'platform': 'android-api-16', 'branch': 'mozilla-central'},
-    {'application': 'fennec', 'platform': 'android-x86', 'branch': 'mozilla-central'},
+    {'application': 'fennec', 'platform': 'android-api-16', 'branch': 'mozilla-central','date':'2019-05-16'},
+    {'application': 'fennec', 'platform': 'android-x86', 'branch': 'mozilla-central','date':'2019-05-16'},
     {'application': 'fennec', 'platform': 'android-api-16',
      'branch': 'mozilla-central', 'date': '2017-08-30'},
```
Comment (whimboo) [inline tests/remote/test_fennec.py]: This test data is duplicated now. Lets remove it and only keep the most recent data as changed above.

**Comment by whimboo** [comment]
Looks fine now. I just removed the extra empty line too, and will push it now.

**Pull request merged**